kola/tests/docker/docker.go

```diff
@@ -254,7 +256,12 @@ func dockerNetwork(c cluster.TestCluster) error {
 // Regression test for https://github.com/coreos/bugs/issues/1569 and
 // https://github.com/coreos/docker/pull/31
 func dockerOldClient(c cluster.TestCluster) error {
-	oldclient := "/usr/lib/kola/amd64/docker-1.9.1"
+	arch := "amd64"
+	if _, ok := c.Cluster.(*qemu.Cluster); ok && kola.QEMUOptions.Board != "" {
+		arch = strings.SplitN(kola.QEMUOptions.Board, "-", 2)[0]
+	}
+
+	oldclient := fmt.Sprintf("/usr/lib/kola/%s/docker-1.9.1", arch)
```

Comment:
Okay, updated.  The oldest CoreOS alpha image for arm64 has Docker 1.10.3, so at least we don't have that binary version.